### PR TITLE
403_ajax_handler

### PR DESCRIPTION
**Pull request by MateuszUrbanowicz**

### CODE REVIEW

File: app/Exceptions/Handler.php [hunk 1]
```diff
@@ -59,35 +63,48 @@ public function render($request, Exception $e)
...
+                                'error' => [
+                                    'message' => ($e->getMessage()) ? $e->getMessage() : 'Internal Server Error',
+                                ]
+                            ],
+                            self::SERVER_ERROR
+                        ),
+                        $request
+                    );
+                } else {
+                    return $CORS->addActualRequestHeaders(
```
Comment (AdrianSkierniewski) [inline app/Exceptions/Handler.php:90]: What about 403 production case?

File: app/Exceptions/Handler.php [hunk 2]
```diff
@@ -59,35 +63,48 @@ public function render($request, Exception $e)
...
+            } elseif ($e instanceof AccessForbiddenException) {
                 return $CORS->addActualRequestHeaders(
                     response()->json(
                         [
-                            'code'  => self::SERVER_ERROR,
-                            'error' => [
-                                'message' => ($e->getMessage()) ? $e->getMessage() : 'Internal Server Error',
-                            ]
+                            'code'  => self::FORBIDDEN_ERROR,
+                            'error' => $e->getMessage()
```
Comment (AdrianSkierniewski) [inline app/Exceptions/Handler.php]: ()? $e->getMessage(): 'Forbidden'

File: app/Exceptions/Handler.php (hunk 1 above)
Comment (MateuszUrbanowicz) [inline app/Exceptions/Handler.php:90]: it is working for both production and develop cases

**Comment by MateuszUrbanowicz** [comment]
@AdrianSkierniewski ptal

File: app/Exceptions/Handler.php [hunk 3]
```diff
@@ -48,6 +50,8 @@ public function render($request, Exception $e)
         if ($request->ajax()) {
             /** @var $CORS \Asm89\Stack\CorsService */
             $CORS = app()->make('Asm89\Stack\CorsService');
+
```
Comment (AdrianSkierniewski) [inline app/Exceptions/Handler.php]: We don't need those two empty lines

**Comment by AdrianSkierniewski** [comment]
After removing those two empty lines :+1:

**Comment by ASkierniewska** [comment]
:+1:

**Pull request merged**